src/http/mod.rs

```diff
@@ -113,4 +132,103 @@ impl Spotify {
             url.to_string()
         }
     }
+
+    /// The headers required for authenticated requests to the API
+    fn auth_headers(&self) -> ClientResult<Headers> {
```

Comment:
Could this function mark as `inline` function as the below functions?

Reply:
Not sure if this one should be `inline`. I only added that for *wrapper* functions, but this is an actual function. It contains more code than the wrappers and would be repeated a lot of times by being inline. Just like how `endpoint_url` isn't `inline` either.

Reply:
ok, I get it.